src/defs/server.ts

```diff
@@ -4,6 +4,51 @@ import {LuaFunction, MTAFunction, ScriptSide} from "./defs";
 
 export var ServerDefinitions = new Array<LuaFunction>();
 
+// tmpDef = new MTAFunction;
+// tmpDef.label = "";
+// tmpDef.description = "";
+// tmpDef.returnType = "";
+// tmpDef.args = [""];
+// tmpDef.argDescs = {};
+// tmpDef.scriptSide = ScriptSide.Client;
+// ServerDefinitions.push(tmpDef);
+
+tmpDef = new MTAFunction;
+tmpDef.label = "dbPrepareString";
+tmpDef.description = "This function escapes arguments in the same way as dbQuery, except dbPrepareString returns the query string instead of processing the query. This allows you to safely build complex query strings from component parts and help prevent (one class of) SQL injection.";
+tmpDef.returnType = "string";
+tmpDef.args = ["element databaseConnection", "string query", "[var param1", "var param2 ...]"];
+tmpDef.argDescs = {};
+tmpDef.scriptSide = ScriptSide.Client;
+ServerDefinitions.push(tmpDef);
+
+tmpDef = new MTAFunction;
+tmpDef.label = "resendPlayerACInfo";
+tmpDef.description = "This function will force the specified player to resend their AC info, triggering the onPlayerACInfo event again.";
+tmpDef.returnType = "bool";
+tmpDef.args = ["player thePlayer"];
+tmpDef.argDescs = {};
+tmpDef.scriptSide = ScriptSide.Client;
+ServerDefinitions.push(tmpDef);
+
+tmpDef = new MTAFunction;
+tmpDef.label = "getCommandHandlers";
+tmpDef.description = "[ resource theResource ]";
+tmpDef.returnType = "table";
+tmpDef.args = ["[resource theResource]"];
+tmpDef.argDescs = {};
+tmpDef.scriptSide = ScriptSide.Client;
+ServerDefinitions.push(tmpDef);
+
+var tmpDef = new MTAFunction;
```

Comment:
Why do you declare tmpDef here via now?

Reply:
most stuff I just copy pasted lol
sorry I know nothing about typeScript language